docs/UserGuide.md

```diff
@@ -47,34 +47,39 @@ If you encounter any issues in launching and using the app, feel free refer to t
   * e.g. while `assign INDEX c/CLASS_NAME` is acceptable, `assign c/CLASS_NAME INDEX` is not acceptable.
 * If a parameter is expected only once in a command but you specified it multiple times, the parser takes only the last occurrence of the parameter.
   * e.g. if you specify `p/12341234 p/56785678`, only `p/56785678` will be taken.
-* Extraneous parameters for commands that do not take in parameters (such as `help`, `list`, `exit` and `clear`) are ignored by the parser.
+* Extraneous parameters for commands that do not require them are ignored by the parser.
   * e.g. if the command specifies `help 123`, the parser interprets it as `help`.
+  * e.g. the command `addc` only takes in a class parameter. If the command specifies `addc n/John Doe c/CS1231S s/Lab1`, the parser interprets it as `addc c/CS1231S`.
 
-### Modes
+## Modes
 In TA-Assist, you can switch into a mode called the **focus** mode, which lets you run tasks that are specific to (module) class. Therefore,
 [some commands](#features-available-in-focus-mode) can only be executed when you are in focus mode. Commands that you can run in the default (unfocused) mode can also
 be run in focus mode. On the other hand, commands that are available only in focus mode cannot be executed in the default (unfocused) mode.
 
 Let's first begin with the commands available in the default mode.
 
+## Commands available in default mode
```

Comment:
There was another (closed) issue that got confused here. Commands that are available in the beginning are also available in focus mode. So terming it "commands available in default mode" and "commands available in focus mode" may be a bit confusing. Can you find a way to write this better? 

Also "default mode" is a new term introduced here without prior explanation.

Reply:
I think the first list can be commands for TA-Assist, then the second list can be commands ONLY available in focus mode, then we perhaps put a note that the original list of commands are also available in focus mode.